applications/nrf5340_audio/src/bluetooth/le_audio_cis_gateway.c

```diff
@@ -106,6 +109,42 @@ static int next_free_audio_stream_index_get(uint8_t *index)
 	return -ENOSPC;
 }
 
+/**
+ * @brief  Get the presentation delay for al headsets
+ *
+ * @param  sel_max Selection of the use of the minimum or maximum
+ *                 preferred presentation delay (TRUE = max; FALSE = min)
+ * @param  index   The index of the hesdset to test against
+ * @param  pd      Pointer to the absolute presentation delay for the
```

Comment:
Could you please elaborate on what is meant by "absolute" in this PR? This makes me think of a value which is always positive.

Reply:
Will remove absolute, but this is the minimum and maximum delay the hardware, decoder and buffering can accommodate.  
It should always be positive and the absolute is without doubted/certain and these are values that.